Revit_Engine/Query/PullEdges.cs

```diff
@@ -35,17 +35,22 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Returns true if FilterRequest should pull edges from Revit Element")]
-        [Input("filterRequest", "FilterRequest")]
+        [Description("Returns true if IRequest should pull edges from Revit Element")]
+        [Input("request", "IRequest")]
         [Output("PullEdges")]
-        public static bool PullEdges(this FilterRequest filterRequest)
+        public static bool PullEdges(this IRequest request)
         {
-            if (filterRequest == null)
+            if (request == null)
                 return false;
 
-            if (filterRequest.Equalities.ContainsKey(Convert.FilterRequest.PullEdges))
+            if (!(request is FilterRequest))
+                return false;
```

Comment:
This method needs to be changed to either to accept a new, appropriate implementation of IRequest.

It's particularly important here because the `Equalities` Dictionary property will be soon removed from the FilterRequest.